src/peerconnection.cpp

```diff
@@ -579,6 +579,33 @@ bool PeerConnection::getSelectedCandidatePair(CandidateInfo *local, CandidateInf
 #endif
 }
 
+void PeerConnection::clearStats(){
+auto sctpTransport = std::atomic_load(&mSctpTransport);
+	if (sctpTransport)
+		return sctpTransport->clearStats();	
+}
+
+const size_t PeerConnection::bytesSent() {
+	auto sctpTransport = std::atomic_load(&mSctpTransport);
+	if (sctpTransport)
+		return sctpTransport->bytesSent();
+	return 0;
+}
+
+const size_t PeerConnection::bytesReceived() {
+	auto sctpTransport = std::atomic_load(&mSctpTransport);
+	if (sctpTransport)
+		return sctpTransport->bytesReceived();
+	return 0;
+}
+
+const std::chrono::milliseconds PeerConnection::rtt() {
```

Comment:
It could be more practical to return a `std::optional<std::chrono::milliseconds>` here, this way, we can differenciate a 0 RTT (theoretically it could happen locally) from a non available one which would result in `nullopt`. What do you think?

Also just a quick comment: `const` is superfluous on returned values.